test/integration/consul-container/libs/cluster/container.go

```diff
@@ -232,6 +239,28 @@ func NewConsulContainer(ctx context.Context, config Config, cluster *Cluster, po
 		info.CACertFile = clientCACertFile
 	}
 
+	// TODO: Support gRPC+TLS port.
```

Comment:
I started working on this but couldn't figure out a way to make `UseGRPCWithTLS = true`. Would appreciate some help from the CTIA team here 🙇🏻‍♂️

Reply:
@boxofrad , thanks for tagging us. Sure. Will take a look. (cc @NiniOak , @nfi-hashicorp )